pyblish_lite/model.py

```diff
@@ -629,16 +630,17 @@ def filterAcceptsRow(self, source_row, source_parent):
         """Exclude items in `self.excludes`"""
         model = self.sourceModel()
         item = model.items[source_row]
-
         key = getattr(item, "filter", None)
         if key is not None:
             regex = self.filterRegExp()
             if regex.pattern():
                 match = regex.indexIn(key)
                 return False if match == -1 else True
-
         for role, values in self.includes.items():
             data = getattr(item, role, None)
+            if hasattr(data, '__iter__'):
+                return True if any(value in values for value in data) else False
```

Comment:
Excellent, thanks for confirming this.

In that case, I'm happy with this.

Let me know if there is anything else you'd like to change, otherwise could you increment the patch version number in `version.py` so that I can make a new release?

It would also be fantastic if you could include a small reproducible that others could try to see what the change is, along with a screenshot of a before-after that I could put into the release notes.